library-model/library-model-generator/src/main/java/com/uber/nullaway/libmodel/StubxWriter.java

```diff
@@ -15,10 +15,17 @@
 public final class StubxWriter {
   /**
    * The file magic number for version 0 .astubx files. It should be the first four bytes of any
-   * compatible .astubx file.
+   * compatible .astubx file. Unused field, keeping for reference.
    */
+  @SuppressWarnings("UnusedVariable")
   private static final int VERSION_0_FILE_MAGIC_NUMBER = 691458791;
 
+  /**
+   * The file magic number for version 1 .astubx files. It should be the first four bytes of any
+   * compatible .astubx file.
+   */
+  private static final int VERSION_1_FILE_MAGIC_NUMBER = 481874642;
```

Comment:
@lazaroclapp should we have a separate magic number (for the astubx file format) and version number?